### PR TITLE
#167728008 Setup unit tests

### PR DESCRIPTION
**Pull request by Abobos**
**What does this PR do?**

 Setup Unit Tests

**Description of Task to be completed?**

- Install mocha, chai, and chai-http dependency
- Write unit test to GET route to "/"
- Write a script in package.json to run test

**How should this be manually tested?**

- Open terminal
- Clone this repo
- run ``npm install``
- run ``npm test``

**Any background context you want to provide?**

 [Chai assertion library](https://www.chaijs.com/api/bdd/)

**What are the relevant stories?**

 [#167728008](https://www.pivotaltracker.com/story/show/167728008)

**Screenshots (if appropriate)**
server test js - cyclops-backend - Visual Studio Code 8_19_2019 7_14_49 PM

**Questions:**

None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,10 +14,13 @@
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
     "body-parser": "^1.19.0",
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
     "cors": "^2.8.4",
     "dotenv": "^6.2.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.3.0",
+    "mocha": "^6.2.0",
```
Comment (Wokoro) [inline package.json]: Well done bro.
I think we would need `sinon` packages too, so we can stub external parts of our unit test.

Comment (Wokoro) [inline package.json:25]: Well done bro.
I think we would need `sinon` package too, so we can stub external parts of our unit test.

**Review by dieudonneAwa — APPROVED** [review]

**Review by Wokoro — COMMENTED** [review]
I think we need a package that would enable us `stub` external parts of our unit tests.

File: package.json (hunk 1 above)
Comment (Abobos) [reply package.json:25]: What is Sinon packages?

Comment (Wokoro) [reply package.json:25]: It's a package that allow us to do things like, stubing, spying and mocking when writing unit tests...

Comment (Abobos) [reply package.json:25]: okay...... I will learn about it .... But I what I did would still suffice.. right?

**Review by richienabuk — APPROVED** [review]

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+repo_token: oXbUEfKxrsWp5IKNRE13j9RMBpGnTPM4j
```
Comment (chukwuemekachm) [inline .coveralls.yml]: Can you add this to an `env-var` on the Travis?
We don't want to expose any secrets on our repo.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Looking good @Abobos.
Have a single concern though.

File: .coveralls.yml (hunk 2 above)
Comment (Abobos) [reply .coveralls.yml]: Okay..very cool

**Comment by chukwuemekachm** [comment]
Can you remove the coveralls file, we no longer need it here.

**Review by chukwuemekachm — APPROVED** [review]
LGTM.

**Review by justinefe — APPROVED** [review]

**Pull request merged**